cmd/interop/src/json_details.h

```diff
@@ -118,4 +126,79 @@ NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE(EncryptionTestVector,
                                    sender_data_info,
                                    leaves);
 
+NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE(KeyScheduleTestVector::Epoch,
```

Comment:
I would seriously consider switching to boost::json

Reply:
Any particular reason?

Reply:
Mainly performance.

Reply:
Since this is just for interop testing, I'm not really worried about performance.  It also looks like `boost::json` lacks any sort of easy type conversion of the kind we have here.  So I'm going to pass for now.

Reply:
`boost;:json` uses a different approach : http://vinniefalco.github.io/doc/json/json/usage/conversion.html.